### PR TITLE
Fixed duplicate run when use multiple instance

### PR DESCRIPTION
**Pull request by armada45-pixel**
Fix https://github.com/rayriffy/elysia-rate-limit/issues/42

change type inject Server return value to `server or null`
because typeof elysia.server is `server or null`
https://github.com/rayriffy/elysia-rate-limit/compare/main...armada45-pixel:dev-arm-1?expand=1#diff-416aa897f3ece839c33af4d41fc86ac3e601372ce3f32cbf0bba7d1789dcd37f
```ts
let serverInstance: Server
const app = new Elysia()
serverInstance = app.server as Server <= fix type required
```

### CODE REVIEW

**Review by rayriffy — CHANGES_REQUESTED** [review]
This change failed example `basic.ts`, and `multiInstance.ts`. Please fix those and do a self-test before request for re-review.

Also, please rename your pull request properly.

**Review by rayriffy — APPROVED** [review]

**Review by rayriffy — CHANGES_REQUESTED** [review]
Just did a retest, but unfortunately `./example/multiInstance.ts` example swagger does not work, please look into that.

**Comment by armada45-pixel** [comment]
I don't know why swagger not working, But after [update dependencies](https://github.com/rayriffy/elysia-rate-limit/pull/43/commits/aec14648008202af897ead2756231a018935f555) everything look fine.

**Review by rayriffy — APPROVED** [review]
Thanks for contributing. When creating the next contribution, please always create a new branch to prevent mixed up with unrelated commit from last contribution.

**Comment by armada45-pixel** [comment]
> Thanks for contributing. When creating the next contribution, please always create a new branch to prevent mixed up with unrelated commit from last contribution.

Sorry, I'dont know this gonna happen.

File: src/services/plugin.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export const plugin = (userOptions?: Partial<Options>) => {
 
     // @ts-expect-error somehow qi is being sent from elysia, but there's no type declaration for it
     plugin.onBeforeHandle({ as: options.scoping }, async ({ set, request, query, path, store, cookie, error, body, params, headers, qi, ...rest }) => {
+      console.log('onBeforeHandle')
```
Comment (armada45-pixel) [inline src/services/plugin.ts:29]: Why use `console.log` instead of `logger`

**Pull request merged**